tests/drivers/dma/test_loop_transfer/prj_frdm_k64f.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_DMA=y
+CONFIG_DMA_MCUX=y
```

Comment:
Instead of this, let's add a dma section in Kconfig.defconfig.mk64f12. Can you also update the testcase.ini file so sanitycheck builds this test for frdm_k64f?